### PR TITLE
[docs] Update Slack Installation section with new App Manifest

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Updated the doc with new App Manifest Installation
- Given the Manifest example and link to latest version also

### CODE REVIEW

File: docs/docs-site/content/administrator/configuration/server/_index.md [hunk 1]
```diff
@@ -776,54 +776,73 @@ One of the flows for sharing query/gist links, the main bot logic lies on top of
...
+          - link_shared
+          - message.channels
+      org_deploy_enabled: false
+      socket_mode_enabled: false
+      is_hosted: false
+
+The latest version of the manifest file is checked-in [here](https://github.com/cloudera/hue/tree/master/tools/slack)
+
+#### Changes needed in the YAML Manifest
+Update with your Hue instance hostname in two locations:
```
[the remark this answers is not in the document]
Comment (romainr) [reply docs/docs-site/content/administrator/configuration/server/_index.md:825]: `Update <hue-instance-hostname>`

File: docs/docs-site/content/administrator/configuration/server/_index.md [hunk 2]
```diff
@@ -776,54 +776,73 @@ One of the flows for sharing query/gist links, the main bot logic lies on top of
...
+- Under **unfurl_domains** (e.g. demo.gethue.com)
```
Comment (romainr) [inline docs/docs-site/content/administrator/configuration/server/_index.md:826]: Why e.g. here and not above?

File: docs/docs-site/content/administrator/configuration/server/_index.md [hunk 3]
```diff
@@ -776,54 +776,73 @@ One of the flows for sharing query/gist links, the main bot logic lies on top of
...
+- Under **event_subscriptions**, update **request_url** `https://<hue-instance-hostname>/desktop/slack/events/`
```
Comment (romainr) [inline docs/docs-site/content/administrator/configuration/server/_index.md:827]: `, update` --> `in`

File: docs/docs-site/content/administrator/configuration/server/_index.md [hunk 4]
```diff
@@ -776,54 +776,73 @@ One of the flows for sharing query/gist links, the main bot logic lies on top of
...
-12. Set Slack API endpoint flag to true
-13. Go to the **OAuth & Permissions** page, copy the **Bot User OAuth Token** and update **slack_bot_user_token**
-14. Similarly, go to the **Basic Information** page, copy the **Verification Token** and update **slack_verification_token**
+        [[slack]]
+        is_enabled=true
+        slack_verification_token=<your-slack-verification-token>
+        slack_bot_user_token=<your-slack-bot-user-token>
 
-Now add the Slack app in your desired channels and send some query/gist links!
+Now add the Slack app in your desired channels and send some [query/gist links](https://docs.gethue.com/user/concept/#share-to-slack)!
```
Comment (romainr) [inline docs/docs-site/content/administrator/configuration/server/_index.md:848]: `https://docs.gethue.com` to remove

**Review by romainr — APPROVED** [review]
Then ship it!

File: docs/docs-site/content/administrator/configuration/server/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (romainr) [reply docs/docs-site/content/administrator/configuration/server/_index.md:825]: maybe even better to just say rename the two `demo.gethue.com` with your Hue host (not using any <....>)

File: docs/docs-site/content/administrator/configuration/server/_index.md [hunk 5]
```diff
@@ -776,54 +776,73 @@ One of the flows for sharing query/gist links, the main bot logic lies on top of
 The 'Smart Quering Assistance' block on top of Hue server which driving the other flow for SQL Assistance is currently a work in progress for replying to users asking questions on how to find certain data tables or to query them.
 
 ### Manual Slack App Installation
-Steps/flow to be followed by the Hue user to set up their own Hue Slack app for their workspace.
+Steps to be followed by the Hue user to set up their own Hue Slack app for their workspace.
```
[the remark this answers is not in the document]
Comment (Harshg999) [reply docs/docs-site/content/administrator/configuration/server/_index.md:779]: The app is needed to be created only once, by the Slack workspace admin. Others can simply interact with it in the channels.
Not every user needs to follow these steps. For e.g. in our demo workspace, I created the app and you were able to interact with it without following the steps @romainr !

[the remark this answers is not in the document]
Comment (romainr) [reply docs/docs-site/content/administrator/configuration/server/_index.md:779]: Ok, so please clarify above line so!

**Review by romainr — APPROVED** [review]

File: docs/docs-site/content/administrator/configuration/server/_index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Harshg999) [reply docs/docs-site/content/administrator/configuration/server/_index.md:779]: On it!

**Pull request merged**